client/src/views/Game.vue

```diff
@@ -236,36 +237,6 @@ function copyToClipboard() {
   setTimeout(() => (showCopiedToClipboard.value = false), 3000);
 }
 
-const average = computed(() => {
```

Comment:
Code is no longer needed because logic is implemented on the server and values are passed to the UI.